ide/static/js/content.js

```diff
@@ -242,8 +260,8 @@ class Content extends React.Component {
     Object.keys(positions).forEach(layerId => {
       const layer = net[layerId];
       layer.state = {
-        top: `${250 + 50 * positions[layerId][1]}px`,
-        left: `${20 + 180 * positions[layerId][0]}px`,
+        top: `${height + 65 * positions[layerId][1]}px`,
+        left: `${width + 80 * positions[layerId][0]}px`,
```

Comment:
The height and width are now based on the browser innerHeight and width, 65 and 80 are for relative spacing between nodes. Not quite sure why its still not centring for other resolutions.